src/SarifWeb/SarifWeb.csproj

```diff
@@ -42,8 +42,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.20.0" />
-    <PackageReference Include="Sarif.Multitool.Library" Version="2.4.14" />
+    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.21.0" />
+    <PackageReference Include="Sarif.Multitool.Library" Version="2.4.16" />
```

Comment:
@EasyRhinoMSFT, do we need something other than this to update the Multitool package?

Reply:
Nope this should be it.
Note: using the Package Manager is the best way to go and ensures that everything is updated correctly.

Reply:
Unfortunately, this approach doesn't work in most of the repos for this particular update. :(

Reply:
What is the issue?

Reply:
No issue, this thread started in a Teams chat. I think some of the context got lost.